klasses/models_test.py

```diff
@@ -67,6 +69,43 @@ def test_klass_display_title():
     assert klass_without_dates.display_title == bootcamp_title
 
 
+def test_next_installment():
```

Comment:
no, you are right, but at this point we should first agree on what None means for an installment.